### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VincentQQu**
Add an AAAI2024 paper in Intensity-Image Reconstruction from Events

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,84 +4,108 @@
 ## <a href="https://www.neuropac.info/">NeuroPAC</a>
 
 ## Table of Contents:
-- [Survey paper](#survey_paper)
+- [Event-based Vision Resources](#event-based-vision-resources)
```
Comment (guillermogb) [inline README.md:7]: Something seems to be affecting the entire table of contents. Could you check and restore? thanks.

File: README.md [hunk 2]
```diff
@@ -920,6 +944,9 @@ arXiv, 2023.
 - <a name="Liu23tpami"></a>Liu, S., Dragotti, P.L.,    
 *[Sensing Diversity and Sparsity Models for Event Generation and Video Reconstruction from Events](https://doi.org/10.1109/TPAMI.2023.3278940)*,    
 IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Code](https://github.com/pld-group/V2E2V)
+- <a name="Qu24aaai"></a>Qu, Q., Shen, Y., Chen, X., Chung, Y.Y., Liu, T.,
```
Comment (guillermogb) [inline README.md:947]: The entry looks good. Thanks.

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (VincentQQu) [reply README.md:923]: Apologies for the inconvenience. The issue was caused by the "auto-update Table of Contents" feature in the "Markdown All in One" extension within VSCode. I have now disabled this function. Please review the pull request again. Thank you for bringing this to my attention.

Comment (guillermogb) [reply README.md:923]: :thinking:  maybe it is not a bad feature, but we would have to fine-tune it.

File: README.md [hunk 3]
```diff
@@ -920,6 +920,9 @@ arXiv, 2023.
 - <a name="Liu23tpami"></a>Liu, S., Dragotti, P.L.,    
 *[Sensing Diversity and Sparsity Models for Event Generation and Video Reconstruction from Events](https://doi.org/10.1109/TPAMI.2023.3278940)*,    
 IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Code](https://github.com/pld-group/V2E2V)
+- <a name="Qu24aaai"></a>Qu, Q., Shen, Y., Chen, X., Chung, Y.Y., Liu, T.,    
+*[E2HQV: High-Quality Video Generation from Event Camera via Theory-Inspired Model-Aided Deep Learning](https://arxiv.org/abs/2401.08117)*,
```
Comment (guillermogb) [inline README.md:924]: Is there an official AAAI link already?
If not, we can keep the arXiv one. 
When the official AAAI link (DOI) is available we can update the link in the title and add and arXiv link in [PDF] next to [Code]. Thanks

**Review by guillermogb — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (VincentQQu) [reply README.md:924]: The official AAAI link is not available at the moment. We will continue using the arXiv link for now. As soon as the official link becomes available, I will promptly update the link in the title, and move arXiv link to [PDF].

**Pull request merged**